AmazonChimeSDK/AmazonChimeSDK/audiovideo/AudioVideoConfiguration.swift

```diff
@@ -43,6 +43,13 @@ import Foundation
         self.init(audioMode: audioMode, callKitEnabled: callKitEnabled, enableAudioRedundancy: enableAudioRedundancy, videoMaxResolution: VideoResolution.videoResolutionHD)
     }
 
+    convenience public init(audioVideoConfig: AudioVideoConfiguration, videoMaxResolution: VideoResolution) {
```

Comment:
This `init` looks odd, we are initing `AudioVideoConfig` from another `AudioVideoConfig`. I don't think this `init` will be needed if we address the comment below.

Reply:
Updated `videoMaxResolution` to a variable so that the new `init` is no longer needed